content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md

```diff
@@ -65,7 +65,7 @@ To troubleshoot, make sure that your `_config.yml` file follows these rules:
 
 This error means that one of the pages on your site includes an invalid datetime.
 
-To troubleshoot, search the file in the error message and the file's layouts for calls to any date-related Liquid filters. Make sure that any variables passed into date-related Liquid filters have values in all cases and never pass `nil` or `""`. For more information, see [Filters](https://shopify.dev/docs/api/liquid/filters) in the Liquid documentation.
```

Comment:
@akordowski Howdy! 👋 It looks like an update made by our team is being reverted here - https://github.com/github/docs/commit/25182f431f6d533fd41ce89c4320a7b530d11a22

Can we ensure the update our team made remains intact? 💛